### ISSUE TITLE
module "vtk-viewport" not installed

### ISSUE THREAD

**Issue by qmiwang** [OHIF/react-vtkjs-viewport#96]
I have run the code to install the modules:

`npm install --save react-vtkjs-viewport vtk.js`

However, run the codes of in `example/`, it raises the error `module "vtk-viewport" not installed`.

Shall I download the source code and compile it with yarn? Or how could I install "vtk-viewport". 
Thanks a lot!!!

**Comment by dannyrb** [comment]
Yes. You use yarn to install all of the packages dependencies, develop, build, and run examples.
